content/docs/14-troubleshooting/04-cluster-deployment.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Cluster Deployment"
+metaTitle: "Troubleshooting steps for erros during a cluster deployment"
+metaDescription: "Troubleshooting steps for errors during a cluster deployment."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+The following steps will help you troubleshoot errors in the event issues arise while deploying a cluster.  
+
+
+
+## Scenario - Instances Continuously Delete Every 30 Min
+
+An instance is launched and terminated every 30 minutes prior to completion of its deployment, and the **Events Tab** lists errors with the following message:
+
+<br />
+
+```bash
+Failed to update kubeadmControlPlane Connection timeout connecting to Kubernetes Endpoint
+```
+
+This behavior can occur when Kubernetes services for the launched instance fail to start properly. 
+Common reasons for why a service may fail are:
+
+- The specified image could not be pulled from the image repository.
+- The cloud init process failed.
+
+### Debugging Steps
+
+1. Initiate an SSH session with the Kubernetes instance using the SSH key provided during provisioning, and log in as user `spectro`. If you are initiating an SSH session into an installer instance, log in as user `ubuntu`.
+
+    ```shell
+        ssh --identity_file <_pathToYourSSHkey_> spectro@X.X.X.X 
+    ```
+
+2. Elevate the user access. 
+
+    ```shell
+        sudo -i
+    ```
+3. Verify the kubelet service is operational.
+	```shell
+        systemctl status kubelet.service
+    ```
+5. If the kubelet service works as expected, do the following.  
```

Comment:
I suggest moving non-working kubelet steps ahead of correctly working kubelet steps. 
The step progression would be:
3. Verify the kubelet service is operational.
4. If the kubelet service does not work as expected, ...
5. If the Kubelet service works as expected, do the following: 
6. Check stdout ...

Reply:
✅